src/errors.ts

```diff
@@ -78,3 +78,11 @@ export class MeshNotLoggedInError extends MeshError {
     super(`MeshNotLoggedInError: ${message}`);
   }
 }
+
+export class MeshInvalidTagValueError extends MeshError {
```

Comment:
d: would be nice to have some info about the broken tag there but I guess depending on the platform this is not always possible and must be done on a case to case basis during error handling for the platforms.

Reply:
Yeah, I don't think it makes sense right now. This means we have to maintain in our code what the standards are, and of course these could change on the platforms. I also noticed the CLI error messages themselves do not contain any info on why it is wrong, in case we could delegate such a message upward to the user.